EventRouter/eventrouterconfig/config_test.go

```diff
@@ -0,0 +1,191 @@
+package eventrouterconfig
+
+import (
+	"fmt"
+	"github.com/FactomProject/live-api/EventRouter/log"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"syscall"
+	"testing"
+	"time"
+)
+
+const testConfig = `
+[eventlistenerconfig]
+  bindaddress = "127.0.0.1"
+  port = "8044"
+  protocol = "udp"
+
+[subscriptionapiconfig]
+  bindaddress = "0.0.0.0"
+  port = "8777"
+  schemes = ["HTTP","HTTPS"]
+`
+
+func init() {
+	log.SetLevel(log.D)
+}
+
+func Test(t *testing.T) {
+	files := renameConfigFiles()
+	defer restoreRenamedFiles(files)
+
+	t.Run("TestNoConfigFound", testNoConfigFound)
```

Comment:
config_test.go:44: Expected a config not found error
probably concurrency issue because another test creates a config file

Reply:
Could not reproduce this, but maybe that can happen if you stop a test with IJ's stop button This test was moved the end to make sure the config files are cleaned up.